guides/RELEASE-PROCESS.md

```diff
@@ -130,3 +130,16 @@ Example changes:
 
 - Renamed or removed classes, mixins, functions, placeholders, or global variables.
 - Major visual changes to existing components
+
+# Sketch Library Release Process
+
+1. Download the [Sketch file from GitHub](https://github.com/CMSgov/design-system/blob/master/design-assets/CMS-Design-System-UI-kit.sketch)
+1. Update the CMSDS Sketch file locally with your changes
+1. Update the .xml file in the following ways
```

Comment:
Where can they find the .xml file (link)? How do they edit it - do they download or open in a text editor or edit in Github?